sql/logic_test.go

```diff
@@ -135,43 +134,12 @@ type logicQuery struct {
 	expectedResults []string
 }
 
-// TODO(pmattis): #1961 is adding a similar type to cli/sql.go. Perhaps move
-// this type into the sql or sql/driver packages and export it so that it can
-// be shared.
-type logicValue string
-
-func (v *logicValue) Scan(value interface{}) error {
-	switch t := value.(type) {
-	case nil:
-		*v = "NULL"
-	case bool:
-		*v = logicValue(strconv.FormatBool(t))
-	case int64:
-		*v = logicValue(strconv.FormatInt(t, 10))
-	case float64:
-		*v = logicValue(strconv.FormatFloat(t, 'g', -1, 64))
-	case []byte:
-		*v = logicValue(t)
-	case string:
-		*v = logicValue(t)
-	case time.Time:
-		*v = logicValue(t.String())
-	default:
-		return fmt.Errorf("unexpected type: %T", value)
-	}
-	return nil
-}
-
 // logicTest executes the test cases specified in a file. The file format is
 // taken from the sqllogictest tool
 // (http://www.sqlite.org/sqllogictest/doc/trunk/about.wiki) with various
 // extensions to allow specifying errors and additional options. See
 // https://github.com/gregrahn/sqllogictest/ for a github mirror of the
 // sqllogictest source.
-//
-// TODO(pmattis): We currently cannot run the tests from sqllogictest due to
-// insufficient SQL coverage (e.g. lack of subqueries and aggregation
-// functions). We should work towards fixing that.
```

Comment:
just checking that removing this TODO is not related. You just happened to remove it right.